elements/pfe-sass/mixins/_mixins.scss

```diff
@@ -158,3 +172,198 @@
+    --pfe-expanded--focus:        #{pfe-color(surface--lightest--link--focus)};
+    --pfe-expanded--border-left:  #{pfe-color(ui-link)};
+}
+
+@mixin pfe-trigger-color-dark {
+    --pfe-expanded--main:         transparent;
+    --pfe-expanded--aux:          #{pfe-color(surface--darkest--text)};
+    --pfe-expanded--focus:        #{pfe-color(surface--darker--link--focus)};
+}
+
+@mixin pfe-trigger-color-dark-expanded {
+    --pfe-expanded--main:         #{pfe-color(surface--darker)};
+    --pfe-expanded--aux:          #{pfe-color(surface--darkest--text)};
+    --pfe-expanded--focus:        #{pfe-color(surface--darker--link--focus)};
+    --pfe-expanded--border-left:  #{pfe-color(surface--lightest)};
+}
+
+@mixin pfe-trigger-color-light-borders {
+    border-top-color: #{pfe-color(surface--border--lightest)};
+    border-right-color: #{pfe-color(surface--border--lightest)};
+    border-left-color: transparent;
+}
+
+@mixin pfe-container {
+  position: relative;
+  display: block;
+  width: 100%;
+
+  // Padding 
+  padding: calc(#{pfe-local(Padding--vertical)} / 2)  #{pfe-local(Padding--horizontal)};
+
+  @media (min-width: #{pfe-breakpoint(sm)}) {
+    padding: #{pfe-local(Padding)};
+  }
+}
+
+/// ===========================================================================
+/// Accordion chevrons
+/// ===========================================================================
+
+@mixin pfe-chevron-expanded-false($position: after) {
+    content: "";
+    position: absolute;
+    top: calc((#{pfe-var(container-spacer)} * #{$pfe-expand_button--padding-factor}) + #{$pfe-expand_chevron--placement} );
+    display: block;
+    border-style: solid;
+    border-width: 0 $pfe-expand_chevron--weight $pfe-expand_chevron--weight 0;
+    height: $pfe-expand_chevron--size;
+    width: $pfe-expand_chevron--size;
+    text-align: center;
+    transition: transform 0.15s;
+    transform: rotate(-135deg);
+    @if $position == before {
+      left: #{pfe-var(container-spacer)};
+    }
+    @else {
+      right: #{pfe-var(container-spacer)};
+    }
+}
+
+@mixin pfe-chevron-expanded-true($position: after) {
+    content: "";
+    position: absolute;
+    top: calc((#{pfe-var(container-spacer)} * #{$pfe-expand_button--padding-factor}) + #{$pfe-expand_chevron--placement} );
+    display: block;
+    width: $pfe-expand_chevron--size;
+    height: $pfe-expand_chevron--size;
+    border-style: solid;
+    border-width: $pfe-expand_chevron--weight $pfe-expand_chevron--weight 0 0;
+    text-align: center;
+    border-bottom: 0;
+    transition: all 0.15s;
+    transform: rotate(135deg);
+    @if $position == before {
+      left: #{pfe-var(container-spacer)};
+    }
+    @else {
+      right: #{pfe-var(container-spacer)};
+    }
+}
+
+@mixin pfe-accordion-panel {
+  display: none;
+  overflow: hidden;
+  will-change: height;
+  border-color: transparent;
+}
+
+@mixin pfe-accordion-panel-expanded($theme: light) {
```

Comment:
there's a bug on http://localhost:1234/elements/pfe-content-set/demo/ right now where the accordions with the `color="lightest"` aren't expanding, because  lines 11 & 15 on the accordion-panel.scss are only targeting `color="light"` and 'color="dark"`

I think we need to break apart the functionality piece: `display block; position relative` from the style pieces